Patch-Source: https://codeberg.org/selfisekai/copium/src/tag/135.0/cr134-unbundle-simdutf.patch
Patch-Source: https://codeberg.org/selfisekai/copium/src/tag/135.0/cr135-unbundle-simdutf-v8-fix.patch

It requires also the following in build.rs, which is added by v8-build.patch:

    println!("cargo:rustc-link-lib=dylib=simdutf");
--
From 936fb52f32573aeeb6ff2ed4293c0577747c7237 Mon Sep 17 00:00:00 2001
From: LN Liberda <lauren@selfisekai.rocks>
Date: Thu, 6 Mar 2025 20:26:19 +0100
Subject: [PATCH] unbundle: add simdutf

--- a/vendor/v8/build/linux/unbundle/replace_gn_files.py
+++ b/vendor/v8/build/linux/unbundle/replace_gn_files.py
@@ -67,6 +67,7 @@ REPLACEMENTS = {
     'openh264': 'third_party/openh264/BUILD.gn',
     'opus': 'third_party/opus/BUILD.gn',
     're2': 'third_party/re2/BUILD.gn',
+    'simdutf': 'third_party/simdutf/BUILD.gn',
     'snappy': 'third_party/snappy/BUILD.gn',
     # Use system libSPIRV-Tools in Swiftshader.
     # These two shims MUST be used together.
--- /dev/null
+++ b/vendor/v8/build/linux/unbundle/simdutf.gn
@@ -0,0 +1,20 @@
+# Copyright 2024 The Chromium Authors
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//build/config/linux/pkg_config.gni")
+import("//build/shim_headers.gni")
+
+shim_headers("header") {
+  root_path = "."
+  headers = [ "simdutf.h" ]
+}
+
+pkg_config("system_simdutf") {
+  packages = [ "simdutf" ]
+}
+
+source_set("simdutf") {
+  public_configs = [ ":system_simdutf" ]
+  public_deps = [ ":header" ]
+}
--- a/vendor/v8/v8/BUILD.gn
+++ b/vendor/v8/v8/BUILD.gn
@@ -4916,6 +4916,7 @@ v8_header_set("v8_internal_headers") {
     ":v8_abseil",
     ":v8_heap_base_headers",
     ":v8_libbase",
+    "//third_party/simdutf:simdutf",
   ]
 }
 
